# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pyqt5/gpep517.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- PyQt5-5.15.11/project.py.orig	2024-07-13 12:45:28.944010700 +0300
+++ PyQt5-5.15.11/project.py	2024-08-06 22:53:02.941449196 +0300
@@ -168,7 +168,7 @@
 
         # Automatically confirm the license if there might not be a command
         # line option to do so.
-        if tool == 'pep517':
+        if tool != 'emptytools':
             self.confirm_license = True
 
         self._check_license()
